frontend/src/api/ErrorResponseHandler.ts

```diff
@@ -16,7 +16,7 @@ export function ErrorResponseHandler(error: AxiosError): ErrorResponse {
 				return {
 					statusCode,
 					payload: null,
-					error: data.errorType,
+					error: data,
```

Comment:
why this change is here

Reply:
There is no `data.errorType` in the response object of AxiosError.

Reply:
let's revert this change though it used everywhere